packages/hooks/src/useBackdropClick.ts

```diff
@@ -0,0 +1,66 @@
+import useEventListener from './useEventListener';
+
+export function useBackdropClick(
+  callback: Function,
+  refEl: React.RefObject<HTMLElement>,
+  enabled?: boolean,
+): void;
+export function useBackdropClick(
+  callback: Function,
+  refEls: Array<React.RefObject<HTMLElement>>,
```

Comment:
is there a reason we can't type this as `refElorEls: React.RefObject<HTMLElement> | Array<React.RefObject<HTMLElement>>` instead of overloading the type?

Reply:
Yeah, got a little too fancy here. This doesn't need to be overloaded